### DISCUSSION TITLE
blog/avoid-becoming-a-faint-king

### DISCUSSION THREAD

**Discussion by giscus** [Pathsis/2750-words#11] [Announcements]
# blog/avoid-becoming-a-faint-king

与人交流或阅读书籍是为了获取赞成、获得回响，这对于每个人而言，都是极其平常的事情。正因为其平凡，当反思这个事情时，会有一点点的惊讶。

https://rercel.com/blog/avoid-becoming-a-faint-king

**Comment by noobnooc** [comment]
作者你好。我是在写一篇关于「博客」的博客时，在 Google 搜索「独立博客」进到你这来的。看了你的很多文章都特别喜欢，思考丰富的同时又带有一点幽默。给你留言只是想让你知道有人在默默地关注着，也许能让你更有坚持下去的动力～

**Reply by Pathsis** [reply]
感谢你的关注，这让我有了继续坚持下去的动力！

**Comment by ghost** [comment]
> 想一想我们与其他人交流的真实情况吧。可能我们在学生时代还有一种求教于人的姿态，但是当走出了学校，当逐渐成为一方领域的资深人员、专家学者，我们的姿态就变成了教授于人。教师写论文，想从学界那里获得赞同。如果写不出来文章，就通过教学从学生那里获得赞同。

我认为这和心理学有个词很相似：投射性认同，恶劣来看是pua，乐观来看是共情的体现；还有一个例子是学术界存在同行评审的制度，有互联网回音墙效应的感觉。

> 比如，如果我们自己是爱国主义者，当我们越过防火长城，我们可能拒绝阅读恨国党写的东西。

让我想到另一个视角，那就是当下中国的犬儒主义盛行，“我知道”成了一种文化，很推荐[徐贲：当今中国犬儒社会文化的困境与出路](https://www.aisixiang.com/data/96680.html)，在高校中是“精致的利己主义者”。

最后想补充个建议，除了接触陌生的事物，还有就是对待事物最好有初学者的姿态，以终为始，更少的武断和偏见，换而言之，避免“在自己构建的世界观中迷失”。

非常高兴能阅读到这篇文章，共勉！

**Reply by ghost** [reply]
> 比如，如果我们自己是爱国主义者，当我们越过防火长城，我们可能拒绝阅读恨国党写的东西。

据我观察，这主要是受到意识形态影响，反贼/民族主义/北美觉醒派（极左）/传统左派/新保守主义，可能在这方面更容易找到认同，例如我就是偏向保守的00后，特别抵触“乌粉”，认为这些内容是信息战的一部分，内容失真而且总是伴有西方媒体的色彩，可我无法理解，我经常反问自己自己的认知真的符合现实吗，内心还是对事物对错非常争执，除非是批判并意识到自己错了，才会了解不同的论点，特别是意识形态或伦理问题......

退一步讲，抵触和自己不同论点的机会成本是什么，是无限广阔的知识视野，是自己当下无法直接认知的事物；我有一个习惯，是定时记录下对事物的看法，过段时间再回头看其实很清楚发现自己过去的偏颇和错误。

真的很高兴看到这篇引人启发的文章！

**Reply by Pathsis** [reply]
谢谢回复！没想到这会让你有这样强烈的反响。

你说“退一步讲，抵触和自己不同论点的机会成本是什么，是无限广阔的知识视野，是自己当下无法直接认知的事物”，我是非常赞同的。

不过有个问题，首先，抵触发生之前，我们可能仔细阅读和审视了许多那类观点，没有发现建设性的东西，最后变成，一看到标签连看都不看就直接否定了，但危险就在于此。其次，随着年龄和学识的增长，抱着初学者的形态考察自己陌生或反对的标签的东西，真的非常浪费时间，事实上也很难做到，可能每一次尝试与之认真对话之后，发现还是和之前一样令人失望。

不过，即使有这样的困难和情有可原之处，我在文中提出的两点还是值得成年人审视的：第一，与人交流或阅读目的只是或主要是为了获得赞同或回响；第二，在了解具体内容之前，就否认自己所陌生或反对的领域或观点是正确的。怎样突破这两点，是成年人的智识保持进步的一个关键。

**Comment by Rayme** [comment]
作者你好，我读了你写的数篇文章感到受益良多，非常感谢你的分享。这一篇引起我的共鸣，于是也想说说我的想法。

我现在是一个传统意义上的爱国主义者/马克思主义者，曾经是 @huaixuOvO 提到的那类北美觉醒派。引起这种变化的核心原因是这几年的疫情改变了我看待社会和世界的角度，以及抛去（一定程度的）偏见的结果。

> 我们对每个我们可能碰到的材料先做一番资格审查。如果某些材料的作者与我们的立场对立，如果某些材料的标签或主义正是我们所反对的，我们会在翻看它们之前就判断它们是错误的。

我十分害怕自己会陷入这样的思想钢印，因此我极力避免我平日接触/阅读到的信息均为合乎自己立场或意识形态的，而是接触各类不同立场或意识形态的媒介发布的任何内容，关注并思考其所表达的思想客观事实（当然我也会忍不住批判不符合客观事实的内容）。

然而在实践上，我很难在阅读某一个作者或者媒体发布的内容时忽略其立场，甚至会在开始阅读之前先下意识打上一个标签，然后在阅读的过程中不断印证最初打下的标签。这依然掉进资格审查的陷阱，并且有点难以主动去避免它的发生。后来我尝试调整思考方式，不再去在意我“有没有打标签”这个事实，而是即便我不自主去印证了，也要试着站在对方的立场理解对方的思想。

我很喜欢文中的这句话：

> 尝试冲破一昧获取赞成、获得回响的牢笼。

共勉！

**Comment by lightstang** [comment]
好文，鼓掌
